frontend/package.json

```diff
@@ -17,15 +16,15 @@
     "http-proxy-middleware": "0.21.0",
     "keycloak-js": "15.0.2",
     "leaflet": "1.8.0",
-    "polylabel": "1.1.0",
     "lodash": "4.17.21",
     "moment": "2.29.3",
     "moment-timezone": "0.5.31",
+    "polylabel": "1.1.0",
     "query-string": "6.12.1",
     "react": "16.12.0",
     "react-app-polyfill": "1.0.5",
     "react-bootstrap": "1.0.0",
-    "react-bootstrap-typeahead": "5.1.2",
+    "react-bootstrap-typeahead": "3.4.7",
```

Comment:
The downgrade may not be needed.  It was to match the `react-bootstrap-typeahead` library.  Presently they are mismatched.